### PR TITLE
code review

### PR DESCRIPTION
**Pull request by awilson28**

### CODE REVIEW

File: browser/js/common/directives/sourceaudio/sourceaudio.directive.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ app.directive("sourceaudio", function (VideoFactory, AudioFactory) {
 				fileInput.click();
 			};
 
+			// comments?
```
Comment (awilson28) [inline browser/js/common/directives/sourceaudio/sourceaudio.directive.js]: massive function, lets have an explanation

File: browser/js/common/directives/sourcevids/sourcevids.controller.js [hunk 2]
```diff
@@ -39,6 +40,7 @@ app.controller("SourceVidsCtrl", function($rootScope, $scope, VideoFactory, Inst
                 // videoElement.duration = duration;
                 videoElement.instructions.endTime = duration;
                 // same here as above
+                // Well done!
                 let phase = $rootScope.$$phase;
```
Comment (awilson28) [inline browser/js/common/directives/sourcevids/sourcevids.controller.js:47]: using phase as discussed. nice

File: browser/js/common/directives/videoplayer/videoplayer.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// SO LONG
```
Comment (awilson28) [inline browser/js/common/directives/videoplayer/videoplayer.js:1]: this directive is HUGE. perhaps we can enhance readability by abstracting the events in it into another factory.

**Comment by kriztynna** [comment]
Ayana - Thanks for your feedback! I resolved a couple of small merge conflicts that arose due to changes that we made after you submitted your request. I also added some additional comments where you noted that they were lacking. We haven't yet refactored our larger factories/directives, but we kept your comments for direction on how to go about it too.
Thanks again!

**Pull request merged**